packaging/windows/BUILD.txt

```diff
@@ -0,0 +1,120 @@
+    copy fop.xconf to /etc/fop.conf
+
+    Install PO4A:
+    https://alioth.debian.org/frs/?group_id=30267
+    Extract and copy to your home folder
+    $ perl Build.PL
+    $ ./Build
+    $ ./Build install
+
+    Modify your bash_profile file in your HOME directory
+    # added as per http://wiki.gimp.org/wiki/Hacking:Building/Windows
+    export PREFIX="/mingw64"
+    export LD_LIBRARY_PATH="$PREFIX/lib:$LD_LIBRARY_PATH"
+
+    JAVA_HOME="/C/Program Files (x86)/Java/jre1.8.0_101"
+    export PATH=$JAVA_HOME/bin:$PATH
+
+    #Update your CLASSPATH
+    CLASSPATH=$CLASSPATH:/usr/share/java/saxon/saxon9he.jar:/usr/share/java/docbook-xsl-saxon/saxon653.jar
+    export CLASSPATH
+
+    #Update your PATH for fop
+    PATH=$PATH:/usr/share/java/fop
+
+    #https://www.ict.griffith.edu.au/anthony/info/graphics/imagemagick.hints
+    export MAGICK_HOME="$PREFIX/bin"
+    export MAGICK_CONFIGURE_PATH="$PREFIX/etc/ImageMagick-7"
+    export MAGICK_CODER_MODULE_PATH="$PREFIX/lib/ImageMagick-7.0.3/modules-Q16HDRI/coders"
+
+    execute the following command to actviate  profile changes 
+    $. .bash_profile
+
+    Clone darktable git repository (in this example into ~/src):
+        $ mkdir ~/src
+        $ cd ~/src
+        $ git clone git://github.com/darktable-org/darktable.git
+    
+    Finally build and install darktable:
+        $ mkdir darktablebuild
+        $ cd darktablebuild
+        $ cmake -G "MSYS Makefiles" -DCMAKE_BUILD_TYPE=Release -DPROJECT_VERSION=2.0.6 -DCMAKE_INSTALL_PREFIX=/opt/darktable ../darktable
+        $ cmake --build .
+        $ cmake --build . --target install
+
+        After this darktable will be installed in /opt/darktable directory and can be started by typing /opt/darktable/bin/darktable.exe in MSYS2 MINGW64 terminal.
+
+    For building the usermanual
+        Add -DBUILD_USERMANUAL=ON flag to the cmake command line
+        
+2). Download and install gtk-bundle for Windows and install it
+    https://github.com/tschoonj/GTK-for-Windows-Runtime-Environment-Installer
+    Pls note: you need to run the installer as admin to run the cache generation
+
+    $ gdk-pixbuf-query-loaders --update-cache 
+
+3). TODO: Run install image creating script, it should create darktable-<VERSION>.msi in current (packaging/windows) directory:
+     $ cmake --build . --target package
+
+ISSUES - FIXED
+    * darktablerc config file  is being generated with Windows style line endings ( CRLF \r\n) , and config file reading functions are expecting a UNIX style line ending (only CR \n)
+    same problem with strip_semicolons_from_keymap function
+    * Reading system parameters (memory/CPU) at init - done
+    * Finding OpenCL library - done
+    * Replacing OpenCL symlinks with a filename/MD5tag combination (symlinks require admin priviledges on windows)
+    * Loading libraries / finding plugin library
+    * Removing ws2_32 and msvcrt as not needed
+    * Added LUA (needs a special old, 5.2 version of the package)
```

Comment:
Can't you just pass `DONT_USE_INTERNAL_LUA=Off` to cmake and be done? That way you should be safe wrt. updated Lua requirements in darktable, for example a potential bump to 5.3.